dist/ccNetViz.js

```diff
@@ -5655,6 +5655,8 @@ var _primitiveTools = __webpack_require__(/*! ./primitiveTools */ "./src/primiti
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
+console.log('test watch');
```

Comment:
Hello @brickmaker , I cannot see this console.log anywhere in the src code. Can you please explain this line of code more?

thanks,
Ales

Reply:
@saskaale , sorry, it's because I use it for verify watch is correct but forget to delete it. When I commit, I only checked the source code and deleted it but not regenerate the dist code.
I've regenerated the dist code.